priorites.go

```diff
@@ -11,28 +13,56 @@ type PriorityProperty struct {
 	Description string `json:"description"`
 }
 
-// Priorities repreents the API response from PagerDuty when listing the
-// configured priorities.
-type Priorities struct {
+// ListPrioritiesResponse repreents the API response from PagerDuty when listing
+// the configured priorities.
+type ListPrioritiesResponse struct {
```

Comment:
I could be convinced to not make this change, but it feels like a pretty low cost to help make this area of the code more consistent when we're already introducing breaking changes.

Reply:
Agreed. I'm a big fan of consistency.

Reply:
I've opened #416 as I remember a way to do this in a non-breaking way. :+1: